snuba/datasets/entities/spans.py

```diff
@@ -0,0 +1,88 @@
+from datetime import timedelta
+from typing import Mapping, Sequence
+
+from snuba.clickhouse.columns import UUID, ColumnSet, DateTime, Nested
+from snuba.clickhouse.columns import SchemaModifiers as Modifiers
+from snuba.clickhouse.columns import String, UInt
+from snuba.clickhouse.translators.snuba.mappers import SubscriptableMapper
+from snuba.clickhouse.translators.snuba.mapping import TranslationMappers
+from snuba.datasets.entities import EntityKey
+from snuba.datasets.entity import Entity
+from snuba.datasets.plans.single_storage import SingleStorageQueryPlanBuilder
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_writable_storage
+from snuba.pipeline.simple_pipeline import SimplePipelineBuilder
+from snuba.query.data_source.join import ColumnEquivalence, JoinRelationship, JoinType
+from snuba.query.extensions import QueryExtension
+from snuba.query.processors import QueryProcessor
+from snuba.query.processors.basic_functions import BasicFunctionsProcessor
+from snuba.query.project_extension import ProjectExtension
+from snuba.query.timeseries_extension import TimeSeriesExtension
+
+
+class SpansEntity(Entity):
+    def __init__(self) -> None:
+        storage = get_writable_storage(StorageKey.SPANS)
+
+        super().__init__(
+            storages=[storage],
+            query_pipeline_builder=SimplePipelineBuilder(
+                query_plan_builder=SingleStorageQueryPlanBuilder(
+                    storage=storage,
+                    mappers=TranslationMappers(
+                        subscriptables=[
+                            SubscriptableMapper(None, "tags", None, "tags")
+                        ],
+                    ),
+                ),
+            ),
+            abstract_column_set=ColumnSet(
```

Comment:
This does not include the description field intentionally as we are still debating on whether it should exist in the storage.